src/components/VmDetails/cards/NicsCard/index.js

```diff
@@ -32,40 +32,43 @@ const NicsCard = ({ vm, vnicProfiles, onEditChange }) => {
   const ip4Label = 'IPv4'
   const ip6Label = 'IPv6'
 
+  const idPrefix = 'vmdetail-nics'
```

Comment:
This will also need to be applied (reapplied) depending on when #796 gets merged relative to this PR.